src/rrt/2dplane/2dplane.cpp

```diff
@@ -10,7 +10,7 @@ using namespace std;
 Tree<Vector2d>* RRT::TreeFor2dPlane(
     shared_ptr<StateSpace<Eigen::Vector2d>> stateSpace, Vector2d goal,
     double step) {
-    Tree<Vector2d>* rrt = new Tree<Vector2d>(stateSpace);
+    Tree<Vector2d>* rrt = new Tree<Vector2d>(stateSpace, hash, dimensions);
```

Comment:
Not sure if this is actually used, but if it is this is probably making memory leaks everywhere. Not very important if its never used(I can't find it used anywhere). But can this just be deleted then? If it is used, it would be best to make it a unique_ptr or at least a shared_ptr.